commons-library/pom.xml

```diff
@@ -8,6 +8,12 @@
     <artifactId>commons-library</artifactId>
     <version>1.0-SNAPSHOT</version>
 
+    <parent>
+        <groupId>com.afh.skytouch</groupId>
+        <artifactId>skytouch-test</artifactId>
+        <version>1.0-SNAPSHOT</version>
```

Comment:
Pasa la version de tu app a una maven property en el parent pom.xml

Reply:
Version was removed from the subprojects, but to identify the pom parent it stills needed the pom's version